src/orders/components/OrderRefundDatagrid/datagrid.ts

```diff
@@ -76,22 +80,26 @@ export const createGetCellContent =
       return readonlyTextCell("");
     }
 
-    // TODO: replace with actual status when API available
-    const color = getStatusColor({
-      status: "generic",
-      currentTheme,
-    });
-
     switch (columnId) {
       case "status":
+        const color = getStatusColor({
+          status: getGrantedRefundStatus(rowData.status),
+          currentTheme,
+        });
+
+        const statusMessage = getGrantedRefundStatusMessage(
+          rowData.status,
+          intl,
+        );
+
         return tagsCell(
           [
             {
-              tag: "DRAFT",
+              tag: statusMessage,
               color: color.base,
             },
           ],
-          ["DRAFT"],
+          [statusMessage],
         );
```

Comment:
**thought**: this should eventually be replaced with `pillCell`, but this would require migrating `EventStatus` component as well (for consistency on order details).  Maybe let's wait for designer and decide later on